Units/parser-vim.r/vim-let-in-function.d/expected.tags

```diff
@@ -0,0 +1,8 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
+!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
+!_TAG_PROGRAM_AUTHOR	Universal Ctags Team	//
+!_TAG_PROGRAM_NAME	Universal Ctags	/Derived from Exuberant Ctags/
+!_TAG_PROGRAM_URL	https://ctags.io/	/official site/
+!_TAG_PROGRAM_VERSION	0.0.0	/56612cf/
```

Comment:
You can trim the pseudo tags from the expected.tags.
units scripts automatically trims them when comparing actual output.

BTW, if you want to test pseudo tags, you may have to use tmain.